angular/index.html

```diff
@@ -51,11 +51,11 @@
     </div>
     <div class="section">
         <div class="col">
-            <h3>Strings</h3>
-            <p>Use single quotes for strings.</p>
+            <h3>controllerAs Syntax</h3>
+            <p> It promotes the use of binding to a "dotted" object in the View (e.g. <code>customer.name</code> instead of <code>name</code>), which is more contextual, easier to read, and avoids any reference issues that may occur without "dotting".</p>
```

Comment:
Your right this description isn't the best.  Lets talk more about what we want to acomplish here.
Also lets break these really long lines up by putting some returns. Becuase its HTML we should be fine in doing so. There are going to be times we'll have longer descriptions and having one giant line will not be fun to read changes too.